### PR TITLE
Scalaio 2017

### PR DESCRIPTION
**Pull request by jeantil**
:)

### CODE REVIEW

File: data/conferences.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
       "key": "scalaio",
       "name": "ScalaIO",
       "url": "http://www.scala.io/",
-      "shortDescription": "La conférence dédiée au langage Scala et à son écosystème se déroule à Lyon les 27-28 octobre 2016. Au menu de la programmation fonctionnel des retours d'expérience et surtout des rencontres !",
-      "tags": [ "Scala", "Functionnal", "Java", "JVM" ]
+      "shortDescription": "La conférence dédiée au langage Scala et à son écosystème se déroule à Lyon les 2-3 Novembre 2017. Au menu de la programmation fonctionnel, des retours d'expérience et surtout des rencontres !",
```
Comment (clacote) [inline data/conferences.json]: programmation fonctionnel -> programmation fonctionnelle

**Review by clacote — CHANGES_REQUESTED** [review]
Une typo

File: data/conferences.json (hunk 1 above)
Comment (jeantil) [reply data/conferences.json]: corrigé :)

**Review by clacote — APPROVED** [review]
LGTM. I won't take the responsibility to merge and deploy, though, as I've never done that before and don't have the credentials.

**Pull request merged**